architecture/secure_storage.rst

```diff
@@ -380,6 +388,31 @@ platforms shall use to implement its security logic
    For OP-TEE to be able to write the RPMB key, ``CFG_RPMB_WRITE_KEY=y`` must be
    configured and ``plat_rpmb_key_is_ready()`` must allow it at runtime.
 
+When programming the RPMB key from normal world the RPMB key must be made
+available to that tool. The RPMB key must for security reasons normally not
+be known outside OP-TEE, but an exception might be made in the factory
+during the manufacturing process.
+
+If the HUK is known, the script `scripts/derive_rpmb_key.py`_ can be used to
+derive the RPMB key.
+
+Pros and cons with OP-TEE automatically writing the RPMB key:
+    1. Automatic writing can be triggered after each boot controlled
+       entirely by the normal world, essentially tricking the secure world
+       to reveal the secret RPMB key. Having a separate OP-TEE binary in the
+       factory is not fully secure since it is enough to restore it from
+       from one device to achieve class breakage.
+
+    2. Automatic writing may target the wrong device if there is more than
+       one RPMB since the device name it determined by the probe order. The
+       probe order may differ between boot loader and the kernel or when
+       rebooting. This problem can be addressed by supplying --rpmb-cid and
+       the CID of the MMC device to use as argument.
+
+    3. Automatic writing must be used at the moment if tee-supplicant is
```

Comment:
remove `if`?
"... must be used at the moment tee-supplicant is configured to emulate..."

Reply:
I'm sorry, I don't follow.